### ISSUE TITLE
sonic_installer failed to remove previous image

### ISSUE THREAD

**Issue by stcheng** [Azure/sonic-utilities#64]
```
sonic_installer remove SONiC-OS-20170515.01
Image will be removed, continue? [y/N]: y
Removing image root filesystem...
Traceback (most recent call last):
  File "/usr/bin/sonic_installer", line 9, in <module>
    load_entry_point('sonic-utilities==1.1', 'console_scripts', 'sonic_installer')()
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/sonic_installer/main.py", line 226, in remove
    shutil.rmtree(HOST_PATH + '/' + image_dir)
  File "/usr/lib/python2.7/shutil.py", line 247, in rmtree
    rmtree(fullname, ignore_errors, onerror)
  File "/usr/lib/python2.7/shutil.py", line 247, in rmtree
    rmtree(fullname, ignore_errors, onerror)
  File "/usr/lib/python2.7/shutil.py", line 247, in rmtree
    rmtree(fullname, ignore_errors, onerror)
  File "/usr/lib/python2.7/shutil.py", line 247, in rmtree
    rmtree(fullname, ignore_errors, onerror)
  File "/usr/lib/python2.7/shutil.py", line 247, in rmtree
    rmtree(fullname, ignore_errors, onerror)
  File "/usr/lib/python2.7/shutil.py", line 247, in rmtree
    rmtree(fullname, ignore_errors, onerror)
  File "/usr/lib/python2.7/shutil.py", line 247, in rmtree
    rmtree(fullname, ignore_errors, onerror)
  File "/usr/lib/python2.7/shutil.py", line 241, in rmtree
    fullname = os.path.join(path, name)
  File "/usr/lib/python2.7/posixpath.py", line 80, in join
    path += '/' + b
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 2: ordinal not in range(128)
```

**Comment by lguohan** [comment]
is it because sudo?

**Comment by stcheng** [comment]
should not be. sonic_installer remove requires root by default.

**Comment by jleveque** [comment]
It appears as if the rmtree() call recursed a few levels into `image_dir` and encountered a path with a non-ASCII character in it.

**Comment by lguohan** [comment]
addressed in #239
